### ISSUE TITLE
v1.1.0 - pip causing failure

### ISSUE THREAD

**Issue by akaszynski** [awalsh128/cache-apt-pkgs-action#37]
Hello,

Looks like there's been a regression in `v1.1.0`, please see:
https://github.com/pyvista/pyvista/runs/7497535215?check_suite_focus=true

**Comment by awalsh128** [comment]
Sorry to hear that. I will add these to the specific tests and repatch this version.

**Comment by awalsh128** [comment]
I am seeing a decryption failure in the log. Am I missing something?

`ssl.SSLError: [SSL: DECRYPTION_FAILED_OR_BAD_RECORD_MAC] decryption failed or bad record mac (_ssl.c:2633)`

Is this not a problem in a prior rev?

**Comment by akaszynski** [comment]
> I am seeing a decryption failure in the log. Am I missing something?
> 
> `ssl.SSLError: [SSL: DECRYPTION_FAILED_OR_BAD_RECORD_MAC] decryption failed or bad record mac (_ssl.c:2633)`
> 
> Is this not a problem in a prior rev?

Seems not. I can rerun just to see if it's a fluke.

**Comment by akaszynski** [comment]
Issue persists:
https://github.com/pyvista/pyvista/runs/7510994102?check_suite_focus=true#step:41:193

**Comment by awalsh128** [comment]
So `dpkg -L libdrm-dev` complains it is not installed after `apt-fast install` is ran using your package list. The problematic package is `libosmesa6-dev`. From the actual install output it looks like it doesn't install the dependency it says it is going to use.

```
[apt-fast 21:23:03]Working... this may take a while.
Reading package lists...
Building dependency tree...
Reading state information...
The following additional packages will be installed:
  blt libegl-dev libegl-mesa0 libegl1 libgbm1 libgl-dev libgles-dev libgles1
  libgles2 libglvnd-dev libglx-dev libopengl-dev libopengl0 libosmesa6
  libwayland-server0 pandoc-data tk8.6-blt2.5
Suggested packages:
  blt-demo texlive-latex-recommended texlive-xetex texlive-luatex
  pandoc-citeproc texlive-latex-extra context wkhtmltopdf librsvg2-bin groff
  ghc nodejs php r-base-core libjs-mathjax node-katex tix python3-tk-dbg
The following NEW packages will be installed:
  blt git-restore-mtime libegl-dev libegl-mesa0 libegl1 libgbm1 libgl-dev
  libgl1-mesa-dev libgles-dev libgles1 libgles2 libglvnd-dev libglx-dev
  libopengl-dev libopengl0 libosmesa6 libosmesa6-dev libwayland-server0 pandoc
  pandoc-data python3-tk tk8.6-blt2.5
0 upgraded, 22 newly installed, 0 to remove and 254 not upgraded.
```

I am guessing this is due to the combined dependency graph; the functionality may be fulfilled using another package. I am going to need to refactor how I approach dependencies. Will update the ticket once I have something ready for staging.

As a workaround, you can just add all the deps to that package in your action input.

**Comment by awalsh128** [comment]
Potential fix. Will test more and issue a patch once added regression tests pass on the CI.

**Comment by HassanKadhem** [comment]
@awalsh128 is the fix going to be merged anytime soon ?

**Comment by awalsh128** [comment]
I've been off the grid for a bit. Will merge in the next day or two.

**Comment by awalsh128** [comment]
[v1.1.0 has been patched with fixes.](https://github.com/awalsh128/cache-apt-pkgs-action/releases/tag/v1.1.0)

**Comment by akaszynski** [comment]
Curious, was the tag deleted and then reuploaded? I though v1.1.0 was already out.

**Comment by awalsh128** [comment]
Yes I replaced the tag with itself at a different commit number. Really I should be using a v1 release structure associated with incremented tags. This way consumers would get the bug fix for free if let's say I attached v1.1.1 to v1 instead. For now it is a it funky.
